.gitlab-ci.yml

```diff
@@ -109,6 +109,11 @@ package:linux-debian-arm:
   tags: [go]
   script: go run mage.go -v PackageLinuxDebianArm
 
+package:linux-debian-armv6l:
```

Comment:
I actually didnt test this CI workflow.
The only reason i added this here was to follow the pattern.

Reply:
Seems fine 👍